components/authz-service/storage/v2/postgres/postgres_test.go

```diff
@@ -971,9 +971,11 @@ func TestDeletePolicy(t *testing.T) {
 	cases := map[string]func(*testing.T){
 		"empty database": func(t *testing.T) {
 			ctx := context.Background()
```

Comment:
[nit] This is fine as-is now, but I suppose it might have been more unit-testy to assert the ID (non-)change in one test case per op; not for _every test case_. But I might be wrong 😉